src/lib/core/CHIPTLVReader.cpp

```diff
@@ -1573,5 +1573,31 @@ CHIP_ERROR TLVReader::GetNextPacketBuffer(TLVReader & reader, uintptr_t & bufHan
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR TLVReader::LookForElementWithTag(const uint64_t aTagInApiForm, chip::TLV::TLVReader * apDstReader) const
```

Comment:
`ap` on `apDstReader` looks a little hungarian, prefer `destReader` or simply `dest`

also, since dest cannot be null, please `chip::TLV::TLVReader &`

also: since already in chip::TLV:: can we reduce to `TLVReader &` ?

Reply:
Done, thanks